__tests__/app.test.js

```diff
@@ -85,3 +85,26 @@ describe('GET /api/articles/:article_id', () => {
   //     });
   // });
 });
+describe('GET /api/articles', () => {
+  test.only('200: Responds with all articles in database', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(Array.isArray(body)).toBe(true);
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: